### DISCUSSION TITLE
Missing Cache file after cloning GitHub repo

### DISCUSSION THREAD

**Discussion by curiotiks** [quarto-dev/quarto-cli#2460] [Q&A] [answered]
I'm stuck. Started a Quarto Book on my MacBook (intel) and was playing with publishing it to gh-pages. Cloned the repo when I got home using my Mac Mini (silicon). I had some issue with the clone and tried a bunch of things. But now even if I create an entirely new book in a different directory unrelated to the previous projects or cloned project (Github isn't even involved) I continue to receive the same error when I try to render *anything*. 

```
 NAME@MiniMac boot_test_two % quarto render 
    [1/4] index.qmd
    ERROR: NotFound: No such file or directory (os error 2), remove '/Users/NAME/Library/Caches/quarto/sass/603954f6f730b7a48ae583e90c07e56e.css'
```

I've already tried reinstalling quarto (delete from applications and reinstall fresh copy) and no change. All the same files still render on my laptop, so it surely has to be some issue related to my Mini.

**Comment by cscheid** [comment, marked as answer]
I'm sorry you're running into this issue. This is clearly a bug on our side, but without a simple case for us to reproduce, it'll be hard to fix. Can you try simply removing that `Caches` directory, as a workaround?

**Reply by curiotiks** [reply]
Hey! I'm tickled that I found a bug if it really is! I was wondering about deleting the cache folder, but wanted to see if anyone had a better idea. Can I provide anything that would be helpful?

**Reply by curiotiks** [reply]
Great news! I just deleted the `../Caches/quatro` directory and I can now preview my most recently created book project. Appears fixed! Thank you!

**Reply by dragonstyle** [reply]
From looking at the code in that area, I'm wondering if the issue is actually a failure to compile some SASS/SCSS (and that we're actually throwing an error while trying to clean up after failing to compile something). 

If your project is using any custom scss, would you try removing that and see whether that clears it up? (we ob should be letting the real error through...)

**Reply by curiotiks** [reply]
The entire project is [here](https://github.com/curiotiks/spss_guides)! I can provide anything system related as needed. I don't have anything unique or complicated. This is only a new book project, created per the directions on the site, with a new chapter added. I was messing with the three different means to publish to Github, but that's about it.
